examples/web_agent/README.md

```diff
@@ -0,0 +1,33 @@
+# WEBAGENT
+
```

Comment:
Please add instructions on:

- Prereqs (python version, poetry, etc)
- Installation (assume the person has poetry installed, what do you do with it)
- A sample run since this example has complexity involved